backend.py

```diff
@@ -1238,6 +1238,9 @@ def installExtLinux(mounts, disk, location=constants.BOOT_LOCATION_MBR):
 # mounting and unmounting of various volumes
 
 def mountVolumes(primary_disk, boot_partnum, primary_partnum, logs_partnum, cleanup, target_boot_mode):
+    mounter = DeviceMounter()
```

Comment:
Does this ever get unmounted?

Reply:
No, once you call explicitly `mount` you are done

Reply:
The installer makes some calls to sgdisk after the main RPM installation. Will that not be a problem?

Reply:
I was looking at `setActiveDiskPartition()`. Even if the kernel does not specifically care about flags changes, gidsk might still call the ioctl to re-read the partition table which would presumably cause a warning?

Reply:
Yes, `sgdisk` print a warning message which is harmless. I tested (much less) the downgrade and it succeeded but the downgrade should use `fdisk` which usually give error for disk busy. I'll test again the downgrade also. Maybe added a log message for `sgdisk` stating that an above warning message is not an issue ?

Reply:
The downgrade process has no issues